Deployment/azuredeploy.json

```diff
@@ -92,9 +92,10 @@
             "properties": {
                 "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
                 "siteConfig": {
-                    "defaultDocuments": [],
+                    "defaultDocuments": [
+                    ],
                     "phpVersion": "",
-                    "use32BitWorkerProcess": false,
+                    "use32BitWorkerProcess": true,
```

Comment:
There were some kind of bug involved that caused this if i remember correctly?

Reply:
Yes, it's a temporary fix for this https://github.com/Azure/azure-functions-host/issues/5430